README.md

```diff
@@ -165,6 +168,9 @@ There are a number of commands that must be run to execute a migration:
 - `pr-preview`: Prints the commit message that would be used for each repository without actually creating a PR; uses the `pr_message` hook.
 - `pr`: Creates a PR for each repo with the message generated from the `pr_message` hook.
 - `version`: Prints Shepherd version
+- `create-issue`: Creates issue on the repo with `title`, `issue_message` and `issues_labels`.
```

Comment:
What's the rationale for splitting out `create-issue` and `update-issue` vs. just having a single issue api.  For the latter, if an issue doesn't currently exist, we create it.  Otherwise, we update it.